Directory.Build.props

```diff
@@ -1,7 +1,7 @@
 <Project>
 
   <PropertyGroup>
-    <LangVersion>latest</LangVersion>
+    <LangVersion>7.3</LangVersion>
```

Comment:
Why does this not work with the latest version of C#?

Reply:
The "latest" version of C# isn't a version and should never be used. 7.3 declares a specific version that will work consistently for developers using Visual Studio 2017 version 15.7 and higher. It avoids situations where the IDE and build behaves differently (read: not predictably) for some users.